package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "teamfresh",
   "version": "0.0.1",
-  "homepage": "http://ckres.github.io/teamfresh",
+  "homepage": "https://cyan33.github.io/teamfresh",
```

Comment:
We could switch back later because I just cannot deploy my github pages on `ckres`'s, so I did it that way.